core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -245,18 +247,20 @@ class ReplicaManagerTest {
       
       assertTrue(fetchCallbackFired)
       assertEquals("Should not give an exception", Errors.NONE.code, fetchError)
+      assertNotEquals("Should not return empty response", MessageSet.Empty, fetchedMessages)
       fetchCallbackFired = false
       
       // Fetch a message above the high watermark as a consumer
       rm.fetchMessages(
         timeout = 1000,
         replicaId = -1,
-        fetchMinBytes = 1,
+        fetchMinBytes = 0,
```

Comment:
@srdo is right, if it is also 1 for consumer, then it will end up in purgatory and the callback will not fire. Will change the other as well.